src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -363,7 +364,11 @@ export const BorrowInput = ({
     if (_isSmartContractAddress) return 'trade.errors.smartContractWalletNotSupported'
     if (
       !hasEnoughBalanceForTx ||
-      (isLendingQuoteSuccess && isEstimatedFeesDataSuccess && !hasEnoughBalanceForTxPlusSweep)
+      (isLendingQuoteSuccess &&
+        isEstimatedFeesDataSuccess &&
+        collateralAsset &&
+        isUtxoChainId(collateralAsset.chainId) &&
+        !hasEnoughBalanceForTxPlusSweep)
```

Comment:
Not technically part of this PR featureset but figured out I'd include it here, since I was getting into an insufficient funds error state while trying to test this